src/main/java/io/github/jhipster/online/service/JHipsterService.java

```diff
@@ -62,7 +66,9 @@ public JHipsterService(LogsService logsService, ApplicationProperties applicatio
 
     public void installNpmDependencies(String generationId, File workingDir) throws IOException {
         this.logsService.addLog(generationId, "Installing the JHipster version used by the project");
-        this.runProcess(generationId, workingDir, "npm", "install", "--ignore-scripts", "--package-lock-only");
+        final String os = System.getProperty("os.name");
+        final String command = os.contains("indows") ? NPM_WIN : NPM_NIX;
```

Comment:
@nomuna : Why is the string "indows" instead of "windows" ? 🤔

Reply:
Will still work if it is "Windows" or "windows" :D

Reply:
Besides, this method is only used / called by the test.... Apparently unnecessary even.

For the application to generate a working zip file under windows the _jhipster_ command has to be set to _jhipster.cmd_ 
in _application-*.yml_. 

So if the _installNpmDependencies_ method is necessary I would suggest that _npm_ command path is also made configurable through _application-*.yml_  so that the executable path can be set to _npm.cmd_. 

Then the OS detection code can be removed from the method.